ci/install-package-dependencies.sh

```diff
@@ -86,6 +86,24 @@ echo "$EDATOOLS_REPO" > "$TMPDIR/obs.list"
 sudo mv "$TMPDIR/obs.asc"  /etc/apt/trusted.gpg.d/obs.asc
 sudo mv "$TMPDIR/obs.list" /etc/apt/sources.list.d/edatools.list
 
+# Install gcc-9 and set it as the default, don't interrupt setup on failure
```

Comment:
I'm happy with the version bump but I would prefer it if we made the script fail if installation fails, especially for the gcc install. Falling back silently to an older version of gcc feels a bit dangerous to me. If we aren't happy making gcc 9 installation mandatory then perhaps it would be worth opening a tracking issue and linking to it in the comment here?

Reply:
Thanks, I've changed it back. I think it was reviewed in that configuration so I don't anticipate this invalidate's anyone's review